test/tests/wordpress.js

```diff
@@ -0,0 +1,116 @@
+var assert = require('assert');
+var _ = require('underscore');
+var util = require('../../lib/util');
+var wordpress = require('../../lib/wordpress');
+
+var mockView = function() { return 'No op'; };
+
+module.exports = function() {
+
+  describe('WordPress utils', function() {
+
+    describe('.getPostPreviewView', function() {
+      it('returns false when not in preview mode', function(done) {
+        util.params = {is_preview: false};
```

Comment:
Make sure to reset the `params` object after each test, else it will break any other test that needs the object. See [example here](https://github.com/solidusjs/solidus-client/blob/master/test/tests/view.js#L218-L222).